libraries/standard/mqtt/src/mqtt.c

```diff
@@ -283,14 +292,11 @@ static MQTTPubAckType_t getAckFromPacketType( uint8_t packetType )
             break;
 
         case MQTT_PACKET_TYPE_PUBCOMP:
-            ackType = MQTTPubcomp;
-            break;
-
         default:
-
             /* This function is only called after checking the type is one of
```

Comment:
Update the comment here to reflect the change.

Reply:
Updated